<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-E5GIE34" failures="1" tests="3" name="TestNG_Learning.TestCase3_DependsOn" time="0.006" errors="0" timestamp="2022-04-29T01:41:30 IST" skipped="2">
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="doUserReg3" time="0.002">
    <failure message="Registration failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Registration failed
at org.testng.Assert.fail(Assert.java:99)
at TestNG_Learning.TestCase3_DependsOn.doUserReg3(TestCase3_DependsOn.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doUserReg3 -->
  <system-out>
    <![CDATA[<a href='..\screenshots\Golf Clubsshot2.png' target='_blank'>ScreenShot </a>
</br><a href='..\screenshots\Golf Clubsshot2.png' target='_blank'><img src='..\screenshots\Golf Clubsshot2.png' height=400 width=400> </a>]]>
  </system-out>
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="doLogin3" time="0.000">
    <skipped/>
  </testcase> <!-- doLogin3 -->
  <system-out/>
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="isSkip" time="0.004">
    <skipped/>
  </testcase> <!-- isSkip -->
  <system-out/>
</testsuite> <!-- TestNG_Learning.TestCase3_DependsOn -->
